Sources/SwiftProtobuf/AnyMessageStorage.swift

```diff
@@ -202,7 +205,18 @@ internal class AnyMessageStorage {
       // never inserted.
       break
 
-    case .contentJSON:
+    case .contentJSON(let contentJSON, let options):
+      do {
+        guard let messageType = Google_Protobuf_Any.messageType(forTypeURL: _typeURL) else {
+            throw BinaryEncodingError.anyTranscodeFailure
+        }
+        _ = try unpack(contentJSON: contentJSON,
+                       extensions: SimpleExtensionMap(),
+                       options: options,
+                       as: messageType)
+      } catch {
+        throw BinaryEncodingError.anyTranscodeFailure
+      }
       // contentJSON requires a good URL and our ability to look up
```

Comment:
This seems to be what was lifted above, but lost the comment, so move the comments and don't repeat this check?

Reply:
Ooops.  Good catch.